### PR TITLE
feedback-1573-test

### PR DESCRIPTION
**Pull request by tihuan**
We'd get snapshot that looks like this!

```
exports[`annotations stacked bar graph renders 1`] = `
Array [
  "<div data-flip-config=\\"{&quot;translate&quot;:true,&quot;scale&quot;:true,&quot;opacity&quot;:true}\\" data-portal-key=\\"portal\\" class=\\"categorical__value___2RKaC\\" data-testclass=\\"categorical-row\\" style=\\"padding: 4px 0px 4px 7px; display: flex; align-items: baseline; justify-content: space-between; margin-bottom: 2px; border-radius: 2px;\\"><div style=\\"margin: 0px; padding: 0px; user-select: none; width: 220px; display: flex; justify-content: space-between;\\"><div style=\\"display: flex; align-items: baseline;\\"><label for=\\"value-toggle-checkbox-B cells\\" class=\\"bp3-control bp3-checkbox\\" style=\\"margin: 0px;\\"><input id=\\"value-toggle-checkbox-B cells\\" data-testclass=\\"categorical-value-select\\" data-testid=\\"categorical-value-select-louvain-B cells\\" type=\\"checkbox\\" checked=\\"\\"><span class=\\"bp3-control-indicator\\"></span></label><span class=\\"bp3-popover-wrapper\\"><span class=\\"bp3-popover-target\\"><span data-testid=\\"categorical-value-louvain-B cells\\" data-testclass=\\"categorical-value\\" aria-label=\\"B cells\\" class=\\"\\" tabindex=\\"0\\" style=\\"width: 213px; color: black; font-style: normal; display: inline-block; overflow: hidden; line-height: 1.1em; height: 1.1em; vertical-align: middle; margin-right: 16px;\\"><span style=\\"width: 213px; color: black; font-style: normal; display: flex; overflow: hidden; line-height: 1.1em; height: 1.1em; vertical-align: middle; margin-right: 16px; justify-content: flex-start;\\"><span style=\\"overflow: hidden; text-overflow: ellipsis; white-space: nowrap; flex-shrink: 1; min-width: 5px;\\">B c</span><span style=\\"position: relative; overflow: hidden; white-space: nowrap;\\"><span style=\\"color: transparent;\\">ells</span><span style=\\"position: absolute; right: 0px; color: black;\\">ells</span></span></span></span></span></span></div><span style=\\"flex-shrink: 0;\\"></span></div><div><span><span data-testclass=\\"categorical-value-count\\" data-testid=\\"categorical-value-count-louvain-B cells\\" style=\\"color: black;\\">342</span><svg display=\\"auto\\" style=\\"margin-left: 5px; width: 11px; height: 11px; background-color: rgb(44, 160, 44);\\"></svg></span></div></div>",
  "<div data-flip-config=\\"{&quot;translate&quot;:true,&quot;scale&quot;:true,&quot;opacity&quot;:true}\\" data-flip-id=\\"1\\" data-portal-key=\\"portal\\" class=\\"categorical__value___2RKaC\\" data-testclass=\\"categorical-row\\" style=\\"padding: 4px 0px 4px 7px; display: flex; align-items: baseline; justify-content: space-between; margin-bottom: 2px; border-radius: 2px;\\"><div style=\\"margin: 0px; padding: 0px; user-select: none; width: 220px; display: flex; justify-content: space-between;\\"><div style=\\"display: flex; align-items: baseline;\\"><label for=\\"value-toggle-checkbox-CD14+ Monocytes\\" class=\\"bp3-control bp3-checkbox\\" style=\\"margin: 0px;\\"><input id=\\"value-toggle-checkbox-CD14+ Monocytes\\" data-testclass=\\"categorical-value-select\\" data-testid=\\"categorical-value-select-louvain-CD14+ Monocytes\\" type=\\"checkbox\\" checked=\\"\\"><span class=\\"bp3-control-indicator\\"></span></label><span class=\\"bp3-popover-wrapper\\"><span class=\\"bp3-popover-target\\"><span data-testid=\\"categorical-value-louvain-CD14+ Monocytes\\" data-testclass=\\"categorical-value\\" aria-label=\\"CD14+ Monocytes\\" class=\\"\\" tabindex=\\"0\\" style=\\"width: 213px; color: black; font-style: normal; display: inline-block; overflow: hidden; line-height: 1.1em; height: 1.1em; vertical-align: middle; margin-right: 16px;\\"><span style=\\"width: 213px; color: black; font-style: normal; display: flex; overflow: hidden; line-height: 1.1em; height: 1.1em; vertical-align: middle; margin-right: 16px; justify-content: flex-start;\\"><span style=\\"overflow: hidden; text-overflow: ellipsis; white-space: nowrap; flex-shrink: 1; min-width: 5px;\\">CD14+ M</span><span style=\\"position: relative; overflow: hidden; white-space: nowrap;\\"><span style=\\"color: transparent;\\">onocytes</span><span style=\\"position: absolute; right: 0px; color: black;\\">onocytes</span></span></span></span></span></span></div><span style=\\"flex-shrink: 0;\\"></span></div><div><span><span data-testclass=\\"categorical-value-count\\" data-testid=\\"categorical-value-count-louvain-CD14+ Monocytes\\" style=\\"color: black;\\">475</span><svg display=\\"auto\\" style=\\"margin-left: 5px; width: 11px; height: 11px; background-color: rgb(255, 127, 14);\\"></svg></span></div></div>",
  "<div data-flip-config=\\"{&quot;translate&quot;:true,&quot;scale&quot;:true,&quot;opacity&quot;:true}\\" data-flip-id=\\"2\\" data-portal-key=\\"portal\\" class=\\"categorical__value___2RKaC\\" data-testclass=\\"categorical-row\\" style=\\"padding: 4px 0px 4px 7px; display: flex; align-items: baseline; justify-content: space-between; margin-bottom: 2px; border-radius: 2px;\\"><div style=\\"margin: 0px; padding: 0px; user-select: none; width: 220px; display: flex; justify-content: space-between;\\"><div style=\\"display: flex; align-items: baseline;\\"><label for=\\"value-toggle-checkbox-CD4 T cells\\" class=\\"bp3-control bp3-checkbox\\" style=\\"margin: 0px;\\"><input id=\\"value-toggle-checkbox-CD4 T cells\\" data-testclass=\\"categorical-value-select\\" data-testid=\\"categorical-value-select-louvain-CD4 T cells\\" type=\\"checkbox\\" checked=\\"\\"><span class=\\"bp3-control-indicator\\"></span></label><span class=\\"bp3-popover-wrapper\\"><span class=\\"bp3-popover-target\\"><span data-testid=\\"categorical-value-louvain-CD4 T cells\\" data-testclass=\\"categorical-value\\" aria-label=\\"CD4 T cells\\" class=\\"\\" tabindex=\\"0\\" style=\\"width: 213px; color: black; font-style: normal; display: inline-block; overflow: hidden; line-height: 1.1em; height: 1.1em; vertical-align: middle; margin-right: 16px;\\"><span style=\\"width: 213px; color: black; font-style: normal; display: flex; overflow: hidden; line-height: 1.1em; height: 1.1em; vertical-align: middle; margin-right: 16px; justify-content: flex-start;\\"><span style=\\"overflow: hidden; text-overflow: ellipsis; white-space: nowrap; flex-shrink: 1; min-width: 5px;\\">CD4 T</span><span style=\\"position: relative; overflow: hidden; white-space: nowrap;\\"><span style=\\"color: transparent;\\"> cells</span><span style=\\"position: absolute; right: 0px; color: black;\\"> cells</span></span></span></span></span></span></div><span style=\\"flex-shrink: 0;\\"></span></div><div><span><span data-testclass=\\"categorical-value-count\\" data-testid=\\"categorical-value-count-louvain-CD4 T cells\\" style=\\"color: black;\\">837</span><svg display=\\"auto\\" style=\\"margin-left: 5px; width: 11px; height: 11px; background-color: rgb(31, 119, 180);\\"></svg></span></div></div>",
  "<div data-flip-config=\\"{&quot;translate&quot;:true,&quot;scale&quot;:true,&quot;opacity&quot;:true}\\" data-flip-id=\\"3\\" data-portal-key=\\"portal\\" class=\\"categorical__value___2RKaC\\" data-testclass=\\"categorical-row\\" style=\\"padding: 4px 0px 4px 7px; display: flex; align-items: baseline; justify-content: space-between; margin-bottom: 2px; border-radius: 2px;\\"><div style=\\"margin: 0px; padding: 0px; user-select: none; width: 220px; display: flex; justify-content: space-between;\\"><div style=\\"display: flex; align-items: baseline;\\"><label for=\\"value-toggle-checkbox-CD8 T cells\\" class=\\"bp3-control bp3-checkbox\\" style=\\"margin: 0px;\\"><input id=\\"value-toggle-checkbox-CD8 T cells\\" data-testclass=\\"categorical-value-select\\" data-testid=\\"categorical-value-select-louvain-CD8 T cells\\" type=\\"checkbox\\" checked=\\"\\"><span class=\\"bp3-control-indicator\\"></span></label><span class=\\"bp3-popover-wrapper\\"><span class=\\"bp3-popover-target\\"><span data-testid=\\"categorical-value-louvain-CD8 T cells\\" data-testclass=\\"categorical-value\\" aria-label=\\"CD8 T cells\\" class=\\"\\" tabindex=\\"0\\" style=\\"width: 213px; color: black; font-style: normal; display: inline-block; overflow: hidden; line-height: 1.1em; height: 1.1em; vertical-align: middle; margin-right: 16px;\\"><span style=\\"width: 213px; color: black; font-style: normal; display: flex; overflow: hidden; line-height: 1.1em; height: 1.1em; vertical-align: middle; margin-right: 16px; justify-content: flex-start;\\"><span style=\\"overflow: hidden; text-overflow: ellipsis; white-space: nowrap; flex-shrink: 1; min-width: 5px;\\">CD8 T</span><span style=\\"position: relative; overflow: hidden; white-space: nowrap;\\"><span style=\\"color: transparent;\\"> cells</span><span style=\\"position: absolute; right: 0px; color: black;\\"> cells</span></span></span></span></span></span></div><span style=\\"flex-shrink: 0;\\"></span></div><div><span><span data-testclass=\\"categorical-value-count\\" data-testid=\\"categorical-value-count-louvain-CD8 T cells\\" style=\\"color: black;\\">217</span><svg display=\\"auto\\" style=\\"margin-left: 5px; width: 11px; height: 11px; background-color: rgb(214, 39, 40);\\"></svg></span></div></div>",
  "<div data-flip-config=\\"{&quot;translate&quot;:true,&quot;scale&quot;:true,&quot;opacity&quot;:true}\\" data-flip-id=\\"4\\" data-portal-key=\\"portal\\" class=\\"categorical__value___2RKaC\\" data-testclass=\\"categorical-row\\" style=\\"padding: 4px 0px 4px 7px; display: flex; align-items: baseline; justify-content: space-between; margin-bottom: 2px; border-radius: 2px;\\"><div style=\\"margin: 0px; padding: 0px; user-select: none; width: 220px; display: flex; justify-content: space-between;\\"><div style=\\"display: flex; align-items: baseline;\\"><label for=\\"value-toggle-checkbox-Dendritic cells\\" class=\\"bp3-control bp3-checkbox\\" style=\\"margin: 0px;\\"><input id=\\"value-toggle-checkbox-Dendritic cells\\" data-testclass=\\"categorical-value-select\\" data-testid=\\"categorical-value-select-louvain-Dendritic cells\\" type=\\"checkbox\\" checked=\\"\\"><span class=\\"bp3-control-indicator\\"></span></label><span class=\\"bp3-popover-wrapper\\"><span class=\\"bp3-popover-target\\"><span data-testid=\\"categorical-value-louvain-Dendritic cells\\" data-testclass=\\"categorical-value\\" aria-label=\\"Dendritic cells\\" class=\\"\\" tabindex=\\"0\\" style=\\"width: 213px; color: black; font-style: normal; display: inline-block; overflow: hidden; line-height: 1.1em; height: 1.1em; vertical-align: middle; margin-right: 16px;\\"><span style=\\"width: 213px; color: black; font-style: normal; display: flex; overflow: hidden; line-height: 1.1em; height: 1.1em; vertical-align: middle; margin-right: 16px; justify-content: flex-start;\\"><span style=\\"overflow: hidden; text-overflow: ellipsis; white-space: nowrap; flex-shrink: 1; min-width: 5px;\\">Dendrit</span><span style=\\"position: relative; overflow: hidden; white-space: nowrap;\\"><span style=\\"color: transparent;\\">ic cells</span><span style=\\"position: absolute; right: 0px; color: black;\\">ic cells</span></span></span></span></span></span></div><span style=\\"flex-shrink: 0;\\"></span></div><div><span><span data-testclass=\\"categorical-value-count\\" data-testid=\\"categorical-value-count-louvain-Dendritic cells\\" style=\\"color: black;\\">37</span><svg display=\\"auto\\" style=\\"margin-left: 5px; width: 11px; height: 11px; background-color: rgb(227, 119, 194);\\"></svg></span></div></div>",
  "<div data-flip-config=\\"{&quot;translate&quot;:true,&quot;scale&quot;:true,&quot;opacity&quot;:true}\\" data-flip-id=\\"5\\" data-portal-key=\\"portal\\" class=\\"categorical__value___2RKaC\\" data-testclass=\\"categorical-row\\" style=\\"padding: 4px 0px 4px 7px; display: flex; align-items: baseline; justify-content: space-between; margin-bottom: 2px; border-radius: 2px;\\"><div style=\\"margin: 0px; padding: 0px; user-select: none; width: 220px; display: flex; justify-content: space-between;\\"><div style=\\"display: flex; align-items: baseline;\\"><label for=\\"value-toggle-checkbox-FCGR3A+ Monocytes\\" class=\\"bp3-control bp3-checkbox\\" style=\\"margin: 0px;\\"><input id=\\"value-toggle-checkbox-FCGR3A+ Monocytes\\" data-testclass=\\"categorical-value-select\\" data-testid=\\"categorical-value-select-louvain-FCGR3A+ Monocytes\\" type=\\"checkbox\\" checked=\\"\\"><span class=\\"bp3-control-indicator\\"></span></label><span class=\\"bp3-popover-wrapper\\"><span class=\\"bp3-popover-target\\"><span data-testid=\\"categorical-value-louvain-FCGR3A+ Monocytes\\" data-testclass=\\"categorical-value\\" aria-label=\\"FCGR3A+ Monocytes\\" class=\\"\\" tabindex=\\"0\\" style=\\"width: 213px; color: black; font-style: normal; display: inline-block; overflow: hidden; line-height: 1.1em; height: 1.1em; vertical-align: middle; margin-right: 16px;\\"><span style=\\"width: 213px; color: black; font-style: normal; display: flex; overflow: hidden; line-height: 1.1em; height: 1.1em; vertical-align: middle; margin-right: 16px; justify-content: flex-start;\\"><span style=\\"overflow: hidden; text-overflow: ellipsis; white-space: nowrap; flex-shrink: 1; min-width: 5px;\\">FCGR3A+ </span><span style=\\"position: relative; overflow: hidden; white-space: nowrap;\\"><span style=\\"color: transparent;\\">Monocytes</span><span style=\\"position: absolute; right: 0px; color: black;\\">Monocytes</span></span></span></span></span></span></div><span style=\\"flex-shrink: 0;\\"></span></div><div><span><span data-testclass=\\"categorical-value-count\\" data-testid=\\"categorical-value-count-louvain-FCGR3A+ Monocytes\\" style=\\"color: black;\\">146</span><svg display=\\"auto\\" style=\\"margin-left: 5px; width: 11px; height: 11px; background-color: rgb(140, 86, 75);\\"></svg></span></div></div>",
  "<div data-flip-config=\\"{&quot;translate&quot;:true,&quot;scale&quot;:true,&quot;opacity&quot;:true}\\" data-flip-id=\\"6\\" data-portal-key=\\"portal\\" class=\\"categorical__value___2RKaC\\" data-testclass=\\"categorical-row\\" style=\\"padding: 4px 0px 4px 7px; display: flex; align-items: baseline; justify-content: space-between; margin-bottom: 2px; border-radius: 2px;\\"><div style=\\"margin: 0px; padding: 0px; user-select: none; width: 220px; display: flex; justify-content: space-between;\\"><div style=\\"display: flex; align-items: baseline;\\"><label for=\\"value-toggle-checkbox-Megakaryocytes\\" class=\\"bp3-control bp3-checkbox\\" style=\\"margin: 0px;\\"><input id=\\"value-toggle-checkbox-Megakaryocytes\\" data-testclass=\\"categorical-value-select\\" data-testid=\\"categorical-value-select-louvain-Megakaryocytes\\" type=\\"checkbox\\" checked=\\"\\"><span class=\\"bp3-control-indicator\\"></span></label><span class=\\"bp3-popover-wrapper\\"><span class=\\"bp3-popover-target\\"><span data-testid=\\"categorical-value-louvain-Megakaryocytes\\" data-testclass=\\"categorical-value\\" aria-label=\\"Megakaryocytes\\" class=\\"\\" tabindex=\\"0\\" style=\\"width: 213px; color: black; font-style: normal; display: inline-block; overflow: hidden; line-height: 1.1em; height: 1.1em; vertical-align: middle; margin-right: 16px;\\"><span style=\\"width: 213px; color: black; font-style: normal; display: flex; overflow: hidden; line-height: 1.1em; height: 1.1em; vertical-align: middle; margin-right: 16px; justify-content: flex-start;\\"><span style=\\"overflow: hidden; text-overflow: ellipsis; white-space: nowrap; flex-shrink: 1; min-width: 5px;\\">Megakar</span><span style=\\"position: relative; overflow: hidden; white-space: nowrap;\\"><span style=\\"color: transparent;\\">yocytes</span><span style=\\"position: absolute; right: 0px; color: black;\\">yocytes</span></span></span></span></span></span></div><span style=\\"flex-shrink: 0;\\"></span></div><div><span><span data-testclass=\\"categorical-value-count\\" data-testid=\\"categorical-value-count-louvain-Megakaryocytes\\" style=\\"color: black;\\">14</span><svg display=\\"auto\\" style=\\"margin-left: 5px; width: 11px; height: 11px; background-color: rgb(188, 189, 34);\\"></svg></span></div></div>",
  "<div data-flip-config=\\"{&quot;translate&quot;:true,&quot;scale&quot;:true,&quot;opacity&quot;:true}\\" data-flip-id=\\"7\\" data-portal-key=\\"portal\\" class=\\"categorical__value___2RKaC\\" data-testclass=\\"categorical-row\\" style=\\"padding: 4px 0px 4px 7px; display: flex; align-items: baseline; justify-content: space-between; margin-bottom: 2px; border-radius: 2px;\\"><div style=\\"margin: 0px; padding: 0px; user-select: none; width: 220px; display: flex; justify-content: space-between;\\"><div style=\\"display: flex; align-items: baseline;\\"><label for=\\"value-toggle-checkbox-NK cells\\" class=\\"bp3-control bp3-checkbox\\" style=\\"margin: 0px;\\"><input id=\\"value-toggle-checkbox-NK cells\\" data-testclass=\\"categorical-value-select\\" data-testid=\\"categorical-value-select-louvain-NK cells\\" type=\\"checkbox\\" checked=\\"\\"><span class=\\"bp3-control-indicator\\"></span></label><span class=\\"bp3-popover-wrapper\\"><span class=\\"bp3-popover-target\\"><span data-testid=\\"categorical-value-louvain-NK cells\\" data-testclass=\\"categorical-value\\" aria-label=\\"NK cells\\" class=\\"\\" tabindex=\\"0\\" style=\\"width: 213px; color: black; font-style: normal; display: inline-block; overflow: hidden; line-height: 1.1em; height: 1.1em; vertical-align: middle; margin-right: 16px;\\"><span style=\\"width: 213px; color: black; font-style: normal; display: flex; overflow: hidden; line-height: 1.1em; height: 1.1em; vertical-align: middle; margin-right: 16px; justify-content: flex-start;\\"><span style=\\"overflow: hidden; text-overflow: ellipsis; white-space: nowrap; flex-shrink: 1; min-width: 5px;\\">NK c</span><span style=\\"position: relative; overflow: hidden; white-space: nowrap;\\"><span style=\\"color: transparent;\\">ells</span><span style=\\"position: absolute; right: 0px; color: black;\\">ells</span></span></span></span></span></span></div><span style=\\"flex-shrink: 0;\\"></span></div><div><span><span data-testclass=\\"categorical-value-count\\" data-testid=\\"categorical-value-count-louvain-NK cells\\" style=\\"color: black;\\">64</span><svg display=\\"auto\\" style=\\"margin-left: 5px; width: 11px; height: 11px; background-color: rgb(148, 103, 189);\\"></svg></span></div></div>",
]
`;
```

### CODE REVIEW

File: client/__tests__/e2e/puppeteerUtils.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const puppeteerUtils = (page) => ({
   },
 
   async getAllByClass(testClass) {
-    return page.$$eval(`[data-testclass=${testClass}]`, (eles) => eles.slice());
+    return page.$$(`[data-testclass=${testClass}]`);
```
Comment (seve) [inline client/__tests__/e2e/puppeteerUtils.js:19]: Ah, if I'm trying to just return the elements, there's no need for the `eval` here. Thanks!

File: client/__tests__/e2e/e2eAnnotations.test.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ afterAll(async () => {
 
 describe.each([
   { withSubset: true, tag: "subset" },
-  { withSubset: false, tag: "whole" },
+  // { withSubset: false, tag: "whole" },
```
Comment (seve) [inline client/__tests__/e2e/e2eAnnotations.test.js:24]:
```suggestion
  { withSubset: false, tag: "whole" },
```
Just re-enabling subset

Comment (tihuan) [reply client/__tests__/e2e/e2eAnnotations.test.js]: Oh thanks for catching that 😄 Enabled!

**Comment by tihuan** [comment]
Always happy to help! Ah didn't think of that lol Typically I defer to PR author to merge, but that's just me 😆

**Comment by seve** [comment]
@tihuan Feel free to take it away then :). Once you merge, I'll separate the test commits for a PR into #1548

**Comment by tihuan** [comment]
Oh!! Sorry I meant deferring to the story owner (you) to merge the feedback PR whenever you like 😆 Sorry for the confusion 😂 

I'll merge this one since I got your permission! Thank you 😄 !

**Comment by seve** [comment]
Oh got it! Terminology with nested PR's is a little tricky 😂  Noted for next time

**Pull request merged**